### ISSUE TITLE
Error: Unable to run the browser

### ISSUE THREAD

**Issue by tgrodano** [DevExpress/testcafe#4168]
### What is your Test Scenario?
Just running the steps from TestCafe Getting Started guide, here https://devexpress.github.io/testcafe/documentation/getting-started/
 
### What is the Current behavior?
1) Try to perform the steps in the TestCafe Getting Started guide. 
2) From a Windows command prompt, enter the command: "testcafe ie test1.js"
3) Upon issuing that command, the output appears to be escape codes for coloring the text. This is the output that gets shown in the command prompt window:

[?25l?[2K?[1A?[2K?[G?[36m|?[39m
?[?25l?[2K?[1A?[2K?[G?[36m/?[39m
?[?25l?[2K?[1A?[2K?[G?[36m-?[39m
?[?25l?[2K?[1A?[2K?[G?[36m\?[39m
?[?25l?[2K?[1A?[2K?[G?[36m|?[39m
?[?25l?[2K?[1A?[2K?[G?[36m/?[39m
?[?25l?[2K?[1A?[2K?[G?[36m-?[39m
?[?25l?[2K?[1A?[2K?[G?[36m\?[39m
?[?25l?[2K?[1A?[2K?[G?[36m|?[39m
?[?25l?[2K?[1A?[2K?[G?[36m/?[39m
?[?25l?[2K?[1A?[2K?[G?[36m-?[39m
?[2K?[1A?[2K?[G?[31mERROR ?[39mWas unable to open the browser "chrome:" due to error.

Error: Unable to run the browser. The file at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe does not exist or is not executable.
    at Object.callee$0$0$ (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\lib\api\open.js:137:23)
    at tryCatch (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:72:40)    at Generator.invoke [as _invoke] (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:334:22)
    at Generator.prototype.(anonymous function) [as throw] (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:105:21)    at tryCatch (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:72:40)
    at invoke (C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:146:20)    at C:\Users\UserName\Documents\TestCafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:156:13

 
### What is the Expected behavior?
browser is launched and test runs

### Steps to Reproduce:
Follow steps describe in the TestCafe Getting Started Guide - here: https://devexpress.github.io/testcafe/documentation/getting-started/

 
### Your Environment details: 
* testcafe version:                   1.4.1
* node.js version:                    10.16
* command-line arguments:  testcafe ie test1.js
* browser name and version: IE 11 (11.765.17134.0), Chrome 75.0.3770.142, Edge 42.17134.1.0
* platform and version:          Windows 10 Enterprise (builds 1703 and 1803)
* other:                                   This issue happens on 4 different PC's

-This issue also exists when trying to run on Chrome or Edge.
-Also tried running with command line:
   testcafe "path:`C:\Program Files\Internet Explorer\iexplore.exe'" test1.js
-Also tried running with command line:  
   testcafe "chrome:C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" test1.js
-Also tried running command line:  "testcafe -b", and it found all the installed browsers: chrome, ie, edge

**Comment by AndreyBelym** [comment]
Thank you for your report.  Please install a debug build, run it with the following commands and share its output in this thread:
```
npm i -g https://git.io/fj7CF
set CI=1
testcafe -v
testcafe chrome tests
```

**Comment by tgrodano** [comment]
@AndreyBelym 
Sorry for taking so long to get back to you, was out on vacation.

Here is the output captured from the command prompt window from the steps you described above:

-------------------------------------------------------------------------------------
C:\TestCafe_TM>npm i -g https://git.io/fj7CF
npm WARN deprecated core-js@1.2.7: core-js@<2.6.8 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
C:\Users\mjala\AppData\Roaming\npm\testcafe -> C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\bin\testcafe-with-v8-flag-filter.js

> core-js@2.6.9 postinstall C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

+ testcafe@1.4.0-rc.1
added 427 packages from 295 contributors in 19.947s

C:\TestCafe_TM>set CI=1

C:\TestCafe_TM>testcafe -v
1.4.0-rc.1

C:\TestCafe_TM>testcafe chrome myfirsttest.ts
TCC: (Sys) The system cannot find the file specified.
".\chrome.exe"
[31mERROR [39mWas unable to open the browser "chrome:" due to error.

Error: Unable to run the browser. The file at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe does not exist or is not executable.

Error: Process start /D "C:\Program Files (x86)\Google\Chrome\Application" .\chrome.exe --remote-debugging-port=42318 --user-data-dir=C:\Users\mjala\AppData\Local\Temp\testcafe\chrome-profile-19248LtkTzbyVcHZv --disable-infobars --disable-session-crashed-bubble --no-first-run --new-window http://10.199.70.253:42316/browser/connect/TLLJDtp exited, code 2
    at ChildProcess.<anonymous> (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\lib\utils\exec.js:118:34)
    at ChildProcess.emit (events.js:198:13)
    at ChildProcess.EventEmitter.emit (domain.js:448:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
    at Object.callee$0$0$ (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\lib\api\open.js:137:23)
    at tryCatch (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:72:40)
    at Generator.invoke [as _invoke] (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:334:22)
    at Generator.prototype.(anonymous function) [as throw] (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:105:21)
    at tryCatch (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:72:40)
    at invoke (C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:146:20)
    at C:\Users\mjala\AppData\Roaming\npm\node_modules\testcafe\node_modules\testcafe-browser-tools\node_modules\babel-runtime\regenerator\runtime.js:156:13
    at process._tickCallback (internal/process/next_tick.js:68:7)

[90mType "testcafe -h" for help.[39m

-------------------------------------------------------------------------------------

The first error seems to indicate that the browser does not exist or is not executable, but it does exist, so perhaps there is a problem executing it? **Perhaps a security setting needs to be changed?** 

I put that path to chrome in the command prompt and chrome was launched. Attached is a screenshot showing that chrome is at that path:

![ChromePath](https://user-images.githubusercontent.com/51338057/64197195-68f4ec80-ce53-11e9-8ed0-3629eeb5d924.png)

**Comment by tgrodano** [comment]
More info...

We are using Take Command/4NT for our ComSpec (command prompt). We need this for some of the other scripts that we run. We are using version:  Take Command v12.10.59 x64.  See: https://jpsoft.com/all-downloads/previous-versions.html  (closest available appears to be v12.11)

This may be the key difference as to why we are seeing this issue and others do not.
----------------------------------------------------------

![4NT](https://user-images.githubusercontent.com/51338057/64199889-58e00b80-ce59-11e9-8d75-09d8850668ce.PNG)

![ComSpec](https://user-images.githubusercontent.com/51338057/64199904-609fb000-ce59-11e9-8b4c-587df1eddf62.PNG)

**Comment by miherlosev** [comment]
Hi @tgrodano 

Thank you for the provided details. We will continue to investigate the problem.
